python_modules/jupedsim/jupedsim/models/anticipation_velocity_model.py

```diff
@@ -90,13 +93,29 @@ class AnticipationVelocityModelAgentParameters:
     anticipation_time: float = 1.0
     reaction_time: float = 0.3
 
+    def __init__(
+        self,
+        v0=None,
+        **kwargs,
+    ):
+        """Init dataclass to handle deprecated argument."""
+        if v0 is not None:
+            warnings.warn(
+                "'v0' is deprecated, use 'desired_speed' instead.",
+                DeprecationWarning,
+                stacklevel=2,
+            )
+            self.desired_speed = v0
+
+        self.__dict__.update(kwargs)
+
```

Comment:
The AVM model has not yet been released, you can freely modify the names still. No need to release with any deprecation.

Reply:
True. I remove v0 from avm.